src/Constrain/CommentPublic.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Constrain;
```

Comment:
Did you mean "Constrain**t**"?

Reply:
https://github.com/cycle/docs/blob/master/advanced/constrain.md

Reply:
"Constrain" is a verb. Noun is "constraint".

Reply:
https://github.com/cycle/orm/issues/65

Reply:
I think we need to use `t` in it.

Reply:
We can use "scope" as was suggested in the issue I've created.